lib/order.rb

```diff
@@ -0,0 +1,97 @@
+# allows use of csv file
+require 'csv'
+# allows access to customer.rb
+require_relative 'customer'
+
+# creates Order class
+class Order
+  # reads data from orders.csv file
+  @@order = CSV.read("data/orders.csv")
+
+  # id attribute is read only
+  attr_reader :id
+  # other attributes can be read and written
+  attr_accessor :products, :customer, :fulfillment_status
+
+  # accepts 4 attributes with default status being :pending
+  def initialize(id, products, customer, fulfillment_status = :pending)
+    @id = id.to_i
+    @products = products
+    @customer = customer
+    @fulfillment_status = fulfillment_status
+      # if status doesn't match any of the following
+      # it will raise ArgumentError
+      if fulfillment_status != :pending &&
+        fulfillment_status != :paid &&
+        fulfillment_status != :processing &&
+        fulfillment_status != :shipped &&
+        fulfillment_status != :complete
+        raise ArgumentError
```

Comment:
Nice! Maybe there's a different way to see if the value of `fulfillment_status` is one of many different options rather than checking if it's equal to several different options?